github/Repository.py

```diff
@@ -3215,6 +3216,14 @@ def get_workflow(self, id_or_name):
         :rtype: :class:`github.Workflow.Workflow`
         """
         assert isinstance(id_or_name, int) or isinstance(id_or_name, str), id_or_name
+        if (
+            type(id_or_name).__name__ == "str"
```

Comment:
why not `isinstance`?

Reply:
Because I was very tired and not sure what I was thinking :) 
Thanks for catching it!